MediaGalleryUi/Model/Config.php

```diff
@@ -21,6 +21,11 @@ class Config implements ConfigInterface
      */
     private const XML_PATH_ENABLED = 'system/media_gallery/enabled';
 
+    /**
+     * Route Path to Media Gallery Index Controller.
+     */
+    const MEDIA_GALLERY_INDEX_ROUTE_PATH = 'media_gallery/index/index';
```

Comment:
Please create two private constants instead of one public - there is a possibility the URLs can be different for openDialog and wysiwyg